packages/jupyter-ai-magics/jupyter_ai_magics/utils.py

```diff
@@ -66,7 +66,7 @@ def get_em_providers(
         try:
             provider = model_provider_ep.load()
         except Exception as e:
-            log.error(
+            log.info(
```

Comment:
Can you also remove the `log.error()` call above in `get_lm_providers()`, in the same file?